content/docs/get-started/index.md

```diff
@@ -59,43 +63,39 @@ save(
 )
 ```
 
-Here, we loaded a well-known
-[Iris flower dataset](https://archive.ics.uci.edu/ml/datasets/iris) with
-`scikit-learn` and trained a simple classifier. Instead of pickling the model,
-we persisted it using MLEM's `save` API.
+Here, we loaded a well-known [Iris flower dataset] with scikit-learn and trained
```

Comment:
```suggestion
Here, we loaded a well-known [Iris flower dataset] with scikit-learn and train
```

Reply:
This seems correct atm. We're using simple past tense in most places after code blocks, same as here ("we loaded ... and trained ...") 🙂  Resolivng for now (please lmk if I didn't understand the suggestion).